cmd/hypper/hypper.go

```diff
@@ -114,3 +114,18 @@ func loadReleasesInMemory(actionConfig *action.Configuration) {
 	// Must reset namespace to the proper one
 	mem.SetNamespace(settings.Namespace())
 }
+
+// This re-Inits the config so we can set the Storage Driver namespace to what is obtained
```

Comment:
This comment is super useful, thanks! :D

nit: But I think it's maybe to big. I would prefer if it were on the commit message of the commit that adds the fix, accessible from git blame.

Reply:
I like the one-line comment `// Reinit the config so we set the proper namespace of the Storage Driver` when it's called, with that is obvious that the call has a purpose. But I'm ok either way, it's a nit :).